portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,14 @@
 #greeting-container {
```

Comment:
General rule of thumb when writing CSS: its best to avoid ID selectors. In small instances like this its fine. But when you work on larger projects you could run into specificity issues because rules with an ID automatically override class-based rules. And issues like that tend to be subtle and hard to debug. Instead it's recommended to duplicate ID as a class name.